modules/core/src/webgl/classes/renderbuffer-formats.js

```diff
@@ -7,56 +7,56 @@ const EXT_FLOAT_WEBGL2 = 'EXT_color_buffer_float';
 // const EXT_HALF_FLOAT_WEBGL1 = 'EXT_color_buffer_half_float';
 
 export default {
-  [GL.DEPTH_COMPONENT16]: {}, // 16 depth bits.
-  [GL.DEPTH_COMPONENT24]: {gl2: true},
-  [GL.DEPTH_COMPONENT32F]: {gl2: true},
+  [GL.DEPTH_COMPONENT16]: {bytesPerPixel: 2}, // 16 depth bits.
+  [GL.DEPTH_COMPONENT24]: {gl2: true, bytesPerPixel: 3},
+  [GL.DEPTH_COMPONENT32F]: {gl2: true, bytesPerPixel: 4},
 
-  [GL.STENCIL_INDEX8]: {}, // 8 stencil bits.
+  [GL.STENCIL_INDEX8]: {bytesPerPixel: 1}, // 8 stencil bits.
 
-  [GL.DEPTH_STENCIL]: {},
-  [GL.DEPTH24_STENCIL8]: {gl2: true},
-  [GL.DEPTH32F_STENCIL8]: {gl2: true},
+  [GL.DEPTH_STENCIL]: {bytesPerPixel: 4},
+  [GL.DEPTH24_STENCIL8]: {gl2: true, bytesPerPixel: 4},
+  [GL.DEPTH32F_STENCIL8]: {gl2: true, bytesPerPixel: 5},
 
   // When using a WebGL 1 context, color renderbuffer formats are limited
-  [GL.RGBA4]: {},
-  [GL.RGB565]: {},
-  [GL.RGB5_A1]: {},
+  [GL.RGBA4]: {bytesPerPixel: 2},
+  [GL.RGB565]: {bytesPerPixel: 2},
+  [GL.RGB5_A1]: {bytesPerPixel: 2},
 
   // When using a WebGL 2 context, the following values are available additionally:
-  [GL.R8]: {gl2: true},
-  [GL.R8UI]: {gl2: true},
-  [GL.R8I]: {gl2: true},
-  [GL.R16UI]: {gl2: true},
-  [GL.R16I]: {gl2: true},
-  [GL.R32UI]: {gl2: true},
-  [GL.R32I]: {gl2: true},
-  [GL.RG8]: {gl2: true},
-  [GL.RG8UI]: {gl2: true},
-  [GL.RG8I]: {gl2: true},
-  [GL.RG16UI]: {gl2: true},
-  [GL.RG16I]: {gl2: true},
-  [GL.RG32UI]: {gl2: true},
-  [GL.RG32I]: {gl2: true},
-  [GL.RGB8]: {gl2: true},
-  [GL.RGBA8]: {gl2: true},
+  [GL.R8]: {gl2: true, bytesPerPixel: 1},
```

Comment:
Tables take a fair bit of space in the bundle, I have always had some angst about these. And I believe that key strings i.e. `{bytesPerPixel: ...}` are usually not minified like variable names (worth checking). I'm normally a big proponent of typing things out but in this case maybe something shorter (`bpp`?), or use arrays and destructure on use.

Reply:
I assumed the minification would work if it's not exported externally? Not something I know too well though, so I'll just change it to `bpp`